x-pack/plugins/maps/public/layers/tooltips/es_agg_tooltip_property.ts

```diff
@@ -4,9 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { ESTooltipProperty } from './es_tooltip_property';
+import { AGG_TYPE } from '../../../common/constants';
+import { ITooltipProperty } from './tooltip_property';
+import { IField } from '../fields/field';
+import { IndexPattern } from '../../../../../../../src/plugins/data/public';
```

Comment:
rewrite as `import { IndexPattern } from 'src/plugins/data/public';` to make ts-linting happy.